server/src/services/cronSchedule/cronSchedule.js

```diff
@@ -8,6 +8,38 @@ import { Applicant } from '../../models/Applicant.js';
 import { Transaction } from '../../models/Transaction.js';
 import { createTransaction, eventHandling } from '../checkResult/checkResult.js';
 
+const fetchTransaction = async (txId) => {
+  console.log(`Transaction ID ${txId}`);
+  const response = await axios.get(`${EXPERIENCE_TRANSACTION_URL}/${txId}`, {
+    headers: {
+      'x-trulioo-api-key': truliooApiKey,
+    },
+  });
+
+  const status = response.data ? response.data.status : 'unknown';
+  console.log(`Status: ${status}`);
+
+  return { status, steps: response?.data?.steps };
+};
+
+/**
+ * // TODO: Document what this is doing and why.
+ *
```

Comment:
Parsing the response about the result of processing the document verification transaction.
See: https://gateway-admin.trulioo.com/documentation/embedid#5.-understanding-the-verification-response

Reply:
@fedencer I'm still confused here. You're saying that `Parsing the response about the result of processing the document verification transaction. See: https://gateway-admin.trulioo.com/documentation/embedid#5.-understanding-the-verification-response` should be the JSDoc comment for `getEmbedIDDocVStep`?

It looks like it's searching an array of `steps` and returning one if it finds a match where `transactionType === 'EmbedID'` & `stepName === 'DocVStep'`, but I'm hoping to understand what the relevance of that is and why we need this function.

Actually, is it ever being used anywhere? I can't find a usage of `getEmbedIDDocVStep`.

Thanks!

@sandoche

Reply:
(I'd botched my search in VSC.)

Now I've found where `getEmbedIDDocVStep` gets used: https://github.com/NEARFoundation/trulioo-app/blob/c5b16d3aef76b4ec60c23388dd98d26850b3d50c/server/src/services/cronSchedule/cronSchedule.js#L83

And I see that I created this function during https://github.com/NEARFoundation/trulioo-app/blob/6de7f74370881505fd9159f15de743a311c2851f/server/src/services/cronSchedule/cronSchedule.js when I was splitting a larger function into smaller ones.

But I'm still trying to understand what is going on here so that we can document it better.

@sandoche , I don't think `Check if application is eligible for document verification` should be the JSDoc either because that sounds like it would return a boolean. But here we're returning a `step` object because later we need to call `const { transactionId, transactionRecordId } = step;`.